source/grid/manifold_lib.cc

```diff
@@ -25,6 +25,67 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+
+namespace internal
+{
+  Tensor<1,3> RotateUnitInDirection ( const Tensor<1,3> &u, const Tensor<1,3> &dir)
```

Comment:
maybe instead of making them internal, add to the `Physics` namespace, see https://www.dealii.org/developer/doxygen/deal.II/namespacePhysics_1_1Transformations_1_1Rotations.html

Reply:
Actually, this is quite different from a regular rotation. In fact, this implements the exponential map for a sphere. I changed the name.

Reply:
ok, if you think none of those internal functions are useful outside of this task, I am fine to keep them internal.